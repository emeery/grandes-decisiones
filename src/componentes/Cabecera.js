import React from 'react';
import {NavLink} from 'react-router-dom';
<<<<<<< HEAD
// una nueva nota pendiente, el oscuro fondo tomaba un aspecto lúgubre 
// y la unica fuente de luz que despedía 3#sx1$. dotted edge 

const Cabecera = () => (
    <header>
    <h1>Portafolio</h1>
    <NavLink to='/' activeClassName='is-active' exact={true}>Hogar</NavLink>
    <NavLink to='/portafolio' activeClassName='is-active' exact={true}>Portafolio</NavLink>
    <NavLink to='/contacto'activeClassName='is-active'>Contacto</NavLink>
=======

// <NavLink to="/ayuda" activeClassName='is-active'>Ayuda</NavLink>
const Cabecera = () => (
    <header>
    <h1>Cuales son tus gastos ?</h1>
    <h3>estrategias para el ahorro</h3>
    <NavLink to="/" activeClassName='is-active' exact={true} >Escritorio</NavLink>
    <NavLink to="/crealo" activeClassName='is-active'>Crear Gastos</NavLink>
    
>>>>>>> 0608cb1624129c0297eabba445c5e9bc3f131dc5
    </header>

); export default Cabecera;

