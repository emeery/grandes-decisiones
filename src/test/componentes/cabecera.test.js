import React from 'react';
import { shallow } from 'enzyme';
import Cabecera from '../../componentes/Cabecera';

// .button, #id, h1
test('deberia renderizar la cabecera ', ()=>{
    const envoltura = shallow(<Cabecera/>);
    expect(envoltura).toMatchSnapshot();
});

// const renderer = new ReactShallowRenderer(); 
    // renderer.render(<Cabecera/>);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());