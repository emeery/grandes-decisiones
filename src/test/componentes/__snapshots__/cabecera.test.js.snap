// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deberia renderizar la cabecera  1`] = `
<header>
  <h1>
    Cuales son tus gastos ?
  </h1>
  <h3>
    estrategias para el ahorro
  </h3>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Escritorio
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/crealo"
  >
    Crear Gastos
  </NavLink>
</header>
`;
